src/Login.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { getUserData } from './apiCalls'
+import { Redirect } from 'react-router-dom';
+import './Login.css';
+
+
+class Login extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      email: '',
+      password: '',
+      error: ''
+    }
+  }
+
+  handleInputChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value })
+  }
+
+  handleLogin = (event) => {
+    event.preventDefault();
+    getUserData(this.state.email, this.state.password)
+    .then(user => this.props.setUser(user))
+    .catch(error => alert("NOPE"))
```

Comment:
I know we were just messing around with this at the time, so we should figure out what should be done with our `error` from the `catch`.   Maybe we can use it to update the `Login` state's `error` key.